CMakeLists.txt

```diff
@@ -602,6 +602,33 @@ if (BUILD_CLIENT)
     find_package(OpenGL REQUIRED)
     include_directories(${OPENGL_INCLUDE_DIR})
     set(LIBS_CLIENT ${LIBS_CLIENT} OpenGL::GL)
+
+    if (LINUX)
+        # Set LEGACY OpenGL ABI if the variable is not explictly set.
+        # The backward-incompatible GLVND OpenGL ABI should only be used
+        # by developers who have a good reason to require it.
+        # See: https://github.com/DaemonEngine/Daemon/issues/945#issuecomment-1835091256
+        set(OpenGL_GL_PREFERENCE "LEGACY" CACHE STRING "Linux OpenGL ABI.")
```

Comment:
Doesn't this need to be set *before* find_package(OpenGL)? As I understand it is a variable to control the behavior of the FindOpenGL module.